packages/amplify-e2e-tests/src/__tests__/graphql-v2/searchable-datastore.test.ts

```diff
@@ -26,8 +26,13 @@ describe('transformer model searchable migration test', () => {
   });
 
   afterEach(async () => {
-    await deleteProject(projRoot);
-    deleteProjectDir(projRoot);
+    if (process.env.CIRCLECI) {
+      console.log('Skipping cloud deletion since we are in CI, and cleanup script will delete this stack in cleanup step.');
+      deleteProjectDir(projRoot);
+    } else {
```

Comment:
would kicking off fire and forget deletion make sense instead?

Reply:
i tried this initially, but jest will wait for any open handles to finish before finishing the test;
ie, 
'amplify delete' runs inside of a terminal spawn, which makes it part of the tests (indirectly, because our tests are configured so that these terminal spawns must succeed/exit successfully, otherwise test fails)

I also tried to manually exit/throw & try catch the error, but the spawn that we use is also responsible for reporting the success of the test; so if any error/process exits the spawn early, it will mark the job/test as failed

Reply:
here is the code, this exitHandler is registered on the terminal that executes each amplify command; so if it exists early from any external process it will still throw a failure
![image](https://user-images.githubusercontent.com/110861985/203653687-ccba1803-1131-4947-adce-2285fd50a24a.png)

Reply:
nexpect.ts
![image](https://user-images.githubusercontent.com/110861985/203653854-7f8e0c6c-866f-4c77-9862-0c68834deb92.png)

Reply:
I see, I was hoping that this is just some CFN delete call that could be fired and forgotten.